wsgi.py

```diff
@@ -7,21 +7,30 @@
 
 logger = logging.getLogger(__name__)
 
-__flask_app__ = create_ui_web_app()
+__flask_app__ = None
 
 
 def application(environ, start_response):
     """WSGI application factory."""
     for key, value in environ.items():
+        # Copy string WSGI envrion to os.environ. This is to get apache
```

Comment:
I sort of wonder if we should abandon using apache SetEnv vars to configure flask.

Some other options are env vars on the process or a python config file.

Reply:
I'd be favorable moving towards python config files, since SetEnv vars have been finicky and error prone.